### ISSUE TITLE
Handle unknown segments better

### ISSUE THREAD

**Issue by Jaimies** [Chemaclass/EdifactParser#24]
I feel like the way unknown segments are currently being treated is not the ideal option. 

Maybe, we should call the "unknown" something else, like "other", and maybe we should also put them separately from the grouped segments, i.e.

```
EdifactParser\TransactionMessage Object
(
    groupedSegments: [...],
    otherSegments: [...],
)  
```

@Chemaclass what do you think?

**Comment by Chemaclass** [comment]
I like this idea very much. Go for it, @Jaimies 👍

**Comment by Jaimies** [comment]
Implemented in #30
